tests/test_airnow.py

```diff
@@ -94,7 +99,7 @@ def test_check_zero_utc_offsets(date, bad_utcoffset, request, printer):
                 f"{len(bad_sites)} sites with zero UTC offset and abs(lon) > 20:\n"
             )
             msg += bad_sites.to_string(index=False)
-            printer(msg)
+            logging.info(msg)
```

Comment:
```suggestion
            printer(msg)
```
In my experience, it's more useful being able to see this every time

Reply:
I'd prefer to use the builtin logging